backend-rust/src/graphql_api.rs

```diff
@@ -63,6 +64,11 @@ use tracing::{error, info};
 use transaction::Transaction;
 
 const VERSION: &str = clap::crate_version!();
+/// The minimum supported database schema version for the API.
+/// Fails at startup if any breaking database schema versions have been
+/// introduced since this version.
+pub const SUPPORTED_SCHEMA_VERSION: migrations::SchemaVersion =
+    migrations::SchemaVersion::InitialFirstHalf;
```

Comment:
Maybe move this to the `migrations.rs` file instead, like the `SchemaVersion::LATEST` const, then we have all of it together.